independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/handlers/ClientSendRequestHandler.java

```diff
@@ -391,6 +390,7 @@ private boolean isResponseMultipart(RestClientRequestContext requestContext) {
     }
 
     private void reportFinish(Throwable throwable, RestClientRequestContext requestContext) {
+        //TODO introduce an abstraction layer not to use ServiceInstance directly?
```

Comment:
what do you have in mind?

Reply:
Not sure what @michalszynkiewicz had in mind but as we move the Stork logic out of this class, it would be nice to not call ServiceInstance directly?